### PR TITLE
Proactively load the noticons font from the new stats resource bundle

### LINKED ISSUE #246: Load Noticons font proactively because of bundle changes
The Noticons font isn't being loaded due to the new bundling with Xcode 6.3 and Cocoapods 0.36. Preload the font so the Storyboard doesn't look fugly.

### PR DESCRIPTION
**Pull request by astralbodies**
Closes #246

### CODE REVIEW

File: StatsDemo/StatsDemo/WPSViewController.m [hunk 1]
```diff
@@ -54,7 +54,14 @@ - (void)checkAuthentication
 #warning You have to replace the site ID, timezone & OAuth2 token with one generated by the WP.com API
         oauth2Token = nil;
         siteId = nil;
-        timeZone = [NSTimeZone timeZoneWithName:@"America/Chicago"];
+        oauth2Token = @"cqi5Cbgie)E@p20veB)x6w^jv9p(s%D*FN0%b^Bi@aTaxJpwC8j29A(znBmNXtHS";
```
Comment (jleandroperez) [inline StatsDemo/StatsDemo/WPSViewController.m]: Should this be... here?

Comment (astralbodies) [inline StatsDemo/StatsDemo/WPSViewController.m]: Heck no. Revoking that token and committing a fix.

**Comment by jleandroperez** [comment]
:shipit:

**Comment by astralbodies** [comment]
Thanks @jleandroperez and @sendhil for reviewing this PR and for catching my OAuth2 token in the commits.

**Comment by jleandroperez** [comment]
Anytime @astralbodies, i owe you!!

**Comment by astralbodies** [comment]
I'll just take the cash => https://cash.me/$astralbodies

**Pull request merged**